### DISCUSSION TITLE
Assignments and score

### DISCUSSION THREAD

**Discussion by MartinBilko** [irgroup-classrooms/dis17-2021#27] [Q&A] [answered]
Related to the first assignment: 
Will you upload a table with the matriculation number and score?

If yes: Will there be such a table for the other assignments as well?

Best regards,
Martin Bilko

**Comment by phschaer** [comment, marked as answer]
Of course. The results are here: [assignment-results.md](https://github.com/irgroup-classrooms/dis17-2021/blob/main/assignment-results.md)

**Reply by MartinBilko** [reply]
Thank you very much.
Since I can't see anyone with 20 points in the list, I assume that no one had a perfect submission of 20 points?

Best regards,
Martin Bilko

**Reply by phschaer** [reply]
You got me! :-)

If mixed up points for assignments 1 and 2. I fixed the numbers.
